logger.js

```diff
@@ -49,6 +49,12 @@ Logger.prototype.error = function error(err) {
         self.exitCode = self.exitCode | err.errorCode & 0x7f;
 
         console.error(err.name + ': ' + err.message);
+
+        if (err.message.lastIndexOf('no peer available', 0) === 0) {
+            console.error('With Hyperbahn, this likely means that the service you are trying to reach is not advertising.');
+            console.error('Without Hyperbahn, this likely means that the peer you are trying to reach is not listening.');
```

Comment:
Agreed second scenario would only happen if `-H /host/file` was an empty array. I think there is no "no peer available" outside of Hyperbahn.